src/Command/AnalyseCommand.php

```diff
@@ -77,21 +77,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			throw new \PHPStan\ShouldNotHappenException();
 		}
 
-		try {
-			$inceptionResult = CommandHelper::begin(
-				$input,
-				$output,
-				$paths,
-				$pathsFile,
-				$memoryLimit,
-				$autoloadFile,
-				$configuration,
-				$level,
-				$allowXdebug
-			);
-		} catch (\PHPStan\Command\InceptionNotSuccessfulException $e) {
```

Comment:
a Exception thrown within a command will automatically lead to a exit-code of `1`

Reply:
Hi, I disagree with this change. CommandHelper outputs the error message, InceptionNotSuccessfulException signals the AnalyseCommand to return the exit code 1.

Reply:
This change would lead to a really chaotic output in most cases which isn't needed.

Reply:
> InceptionNotSuccessfulException signals the AnalyseCommand to return the exit code 1.

symfony will automatically use a exit-code of 1 when a un-handled exception occurs.
no need to do this here and interrupt the exception handling

Reply:
> This change would lead to a really chaotic output in most cases which isn't needed.

IMO the cause of chaotic output is that the output is scattered accross CommandHelper and the actual commands

Reply:
But the problem is that I don't always want to output the thrown exception with stack trace. I want to present something more friendly to the user.

Reply:
when letting the exception fly through we would get a error like

![image](https://user-images.githubusercontent.com/47448731/68298524-e2020180-0099-11ea-81d4-0ad10524742f.png)

no stacktrace or similar involved.

you get the stacktrace etc. when adding `-v` to the command